index.js

```diff
@@ -99,11 +132,17 @@ CachingWriter.prototype._conditionalBuild = function () {
         return rimraf(writer.outputPath);
       }).then(function() {
         fs.mkdirSync(writer.outputPath);
-      });
+      }).finally(function() {
+
+        this.debug('purge output in %dms', new Date() - start);
+        start = new Date();
```

Comment:
I could be misinterpreting this, but `start` was defined above (before the promise) and is being reset here in the `finally`.  Won't this make the later `debug` call that does `new Date() - start` just show the time from when this finally is called and the next finally is called?